protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/record/value/deployment/DecisionRequirementsMetadataRecord.java

```diff
@@ -37,6 +37,7 @@ public class DecisionRequirementsMetadataRecord extends UnifiedRecordValue
   private final BinaryProperty checksumProp = new BinaryProperty("checksum");
 
   private final BooleanProperty isDuplicateProp = new BooleanProperty("isDuplicate", false);
+  private final StringProperty tenantIdProp = new StringProperty("tenantId", "");
```

Comment:
❓ Should the default be `<default>` instead?

Reply:
This will depend on our approach to postponing the migration tasks.

🔧 We could extract a TenantIdPropertyFactory or so, and reuse this in all records so we can easily control the default in one spot

Reply:
I think it would be safer, for the sake of validation to leave it empty.

We have the following cases:
* Multi-tenancy disabled:
   * Users can leave that property empty in their gRPC requests. Internally, Zeebe maps all data to the `<default>` tenant ID.
* Multi-tenancy enabled:
   * Users must define the `tenantId` property in their gRPC requests. Zeebe should reject requests/commands that don't set this property.

In the future, we will expose an API/interface in the Gateway so users may replace the `IdentityInterceptor` with their own implementation. We **might** also allow users to replace the `<default>` tenant ID with something else. So it would be better if we don't hard-code it in the records.